GraphicEditor/View/UserControls/HospitalMapUserControl.xaml.cs

```diff
@@ -11,5 +13,23 @@ public HospitalMapUserControl()
         {
             InitializeComponent();
         }
+
+        //private void CommandBinding_Executed(object sender, ExecutedRoutedEventArgs e)
```

Comment:
What is purpose of this chunk of code?

Reply:
Same answer as before. It is the help pattern.